src/reuse/modules/util/userSettings.ts

```diff
@@ -0,0 +1,94 @@
+"use strict";
+
+import { VerboseLoggerFactory } from "../../helper/verboseLogger";
+
+/**
+ * @class userSettings
+ * @memberof util
+ */
+
+export class UserSettings {
+  private vlf = new VerboseLoggerFactory("util", "user");
+  private _srvInstance = null;
+
+  private async _initForUserSetting(user: string, password: string): Promise<any> {
+    const vl = this.vlf.initLog(this._initForUserSetting);
+    //TODO: Add vl.log msg
+    const params = browser.config.params;
+    if (params?.systemUrl) {
+      this._srvInstance = await service.odata.init(`${params.systemUrl}/sap/opu/odata/UI2/INTEROP`, user, password);
```

Comment:
browser.config.params.odataUrl can not be taken since the default for odataUrl is
`/sap/opu/odata/sap` and the `INTEROP`service url is `/sap/opu/odata/UI2/INTEROP/` so the automatic concat of `/sap/opu/odata/sap/UI2/INTEROP`would lead to an incorrect URL